### DISCUSSION TITLE
Don't have AMD PBS, AMD CBS and other options

### DISCUSSION THREAD

**Discussion by Fetnak** [DavidS95/Smokeless_UMAF#36] [Q&A]
When I open "Device Manager" I receive only "Network Device List" option. There is no AMD PBS, AMD CBS or other. I tried beta UMAF too. Did HP somehow hide this options? Or maybe they are using BIOS thats not supported by UMAF. Can you help me with this?

Laptop: HP ProBook 445 G8
CPU: Ryzen 5 5600U

**Comment by lenielsen** [comment]
Same with Probook G9 and Elitebook G10..
https://github.com/DavidS95/Smokeless_UMAF/discussions/38#discussioncomment-6846661

**Comment by onyxiarivera** [comment]
Same issue with HP ProBook 445 G7
